docs/user/openstack/customization.md

```diff
@@ -23,6 +23,7 @@ Beyond the [platform-agnostic `install-config.yaml` properties](../customization
 * `externalDNS` (optional list of strings): The IP addresses of DNS servers to be used for the DNS resolution of all instances in the cluster
 * `externalNetwork` (optional string): Name of external network the installer will use to provide access to the cluster. If defined, a floating ip from this network will be created and associated with the bootstrap node to facilitate debugging and connection to the bootstrap node during installation. The lbFloatingIP property is a floating ip address selected from this network.
 * `lbFloatingIP` (optional string): Address of existing Floating IP from externalNetwork the installer will associate with the API load balancer. This property is only valid if externalNetwork is defined. If externalNetwork is not defined, this property is ignored.
```

Comment:
We actually error out now when the externalNetwork is not defined. Let's update the doc.